pyproject.toml

```diff
@@ -79,6 +80,9 @@ build-backend = "poetry.core.masonry.api"
 "mlip.md_parser" = "aiida_mlip.parsers.md_parser:MDParser"
 "mlip.train_parser" = "aiida_mlip.parsers.train_parser:TrainParser"
 
+[tool.poetry.plugins."aiida.workflows"]
+"mlip.hts_wg" = "aiida_mlip.workflows.hts_workgraph:HTSWorkGraph"
+
```

Comment:
We are working on a new entry point for the WorkGraph. Will let you know the syntax when the API is fixed.